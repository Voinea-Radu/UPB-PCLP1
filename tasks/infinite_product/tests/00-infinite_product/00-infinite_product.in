9
1 1
2 2
2 4
2 4
2 5
2 5
3 5
3 2
12 12
