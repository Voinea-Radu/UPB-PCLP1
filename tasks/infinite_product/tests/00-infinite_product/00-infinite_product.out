162
3 5
11.7898261 14.6969385
